### PR TITLE
[LW-8984] Accept handles with char matching @

### PR DESCRIPTION
**Pull request by VanessaPC**
# Context
As a phase 1 of the ADA handle support, we didn't support @ symbols, that were used in sub handles. Currently we will be supporting the character in preparation for further support. 

# Proposed Solution
Allow the projection to add/return this symbols 

# Important Changes Introduced

### CODE REVIEW

File: packages/core/src/Asset/util/isValidHandle.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export const isValidHandle = (handle: string) => {
-  const pattern = /^[\w.-]+$/;
+  const pattern = /^[\w.-\s@]+$/;
```
Comment (iccicci) [inline packages/core/src/Asset/util/isValidHandle.ts:2]: Added `\s` as well. Doesn't this means accepting tabs and new lines as well?
I would increase the strength of the tests adding a handle with a space in `invalidHandles` (I guess)

**Review by iccicci — CHANGES_REQUESTED** [review]

**Review by gytis-ivaskevicius — DISMISSED** [review]
Do we have a spec or some definition which lists all supported characters? Maybe there are more edge cases like this

**Comment by gytis-ivaskevicius** [comment]
Actually, isn't it being validated on blockchain already? If that's the case maybe we can remove validation from our side?

File: packages/core/src/Asset/util/isValidHandle.ts (hunk 1 above)
Comment (VanessaPC) [reply packages/core/src/Asset/util/isValidHandle.ts:2]: you are right, it does support tabs and new lines. Will re do it, and will add to the tests.

**Comment by VanessaPC** [comment]
@gytis-ivaskevicius  we are taking the rules from [here](https://docs.adahandle.com/best-practices). 
@iccicci  we are using the handle for the validation of inputs in Lace.

**Review by iccicci — APPROVED** [review]
LGTM!

**Pull request closed without merging**